### PR TITLE
Installing python library.

### PR DESCRIPTION
**Pull request by fabianbl**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -14,8 +14,8 @@ ExternalProject_Add(protobuf_src
   PATCH_COMMAND patch -p0 < ${CMAKE_SOURCE_DIR}/unused_parameter_warnings.patch &&
                 patch -p0 < ${CMAKE_SOURCE_DIR}/unused_parameter_warnings_wire_format.patch
   CONFIGURE_COMMAND cd ../protobuf_src && ./autogen.sh && ./configure --with-pic --prefix=${CATKIN_DEVEL_PREFIX}
-  BUILD_COMMAND cd ../protobuf_src && make -j8
-  INSTALL_COMMAND cd ../protobuf_src && make install -j8
+  BUILD_COMMAND cd ../protobuf_src && make -j8 && cd python && python setup.py build --build-purelib build
+  INSTALL_COMMAND cd ../protobuf_src && make install -j8 && pwd && cp -a python/build/google ${CATKIN_DEVEL_PREFIX}/${CATKIN_GLOBAL_PYTHON_DESTINATION}
```
Comment (mbuerki) [inline CMakeLists.txt]: what is `pwd` used for for?

Comment (fabianbl) [reply CMakeLists.txt]: ups, `pwd` was just there for testing :-)

@schneith Yes, that would definitely be the better option, but I could not find the proper command. `python setup.py install` has e.g. a `--root` option, but then it still installs everything to `usr/local/lib/python2.7/dist-packages` and I was not able to change this behavior. Will try again to do so though.

Comment (schneith) [reply CMakeLists.txt]: Im also fine with the current implementation if the setuptools method takes too much time to get it working.

**Comment by fabianbl** [comment]
@mbuerki I need it for unit tests that import protobuf generated files. Another option would be to install it with pip are apt-get, but it's cleaner to have the version that the protobuf files were generated with.

**Comment by fabianbl** [comment]
test this please

**Comment by mbuerki** [comment]
@fabianbl sounds good!

File: CMakeLists.txt (hunk 1 above)
Comment (fabianbl) [reply CMakeLists.txt]: @schneith I will check, I would also prefer the setuptools option.

**Comment by fabianbl** [comment]
Somehow Jenkins is not working correctly... Here again the working build for the current version: https://jenkins.asl.ethz.ch/job/protobuf_catkin_multiarch/32/

**Pull request merged**